scripts/ci.sh

```diff
@@ -39,7 +39,7 @@ elif [ "$GETH" = true ]; then
   run_geth
   sleep 30
   lerna run --scope truffle test --stream -- --exit
-  lerna run --scope truffle-contract test --stream -- --exit
+  lerna run --scope contract test --stream -- --exit
```

Comment:
```suggestion
  lerna run --scope @truffle/contract test --stream -- --exit
```

Reply:
Is this not a reference to the directory and not the package?

Reply:
I think it's supposed to be a reference to the package (at least that's what the build failure seemed to suggest)

Reply:
I thought so! https://github.com/trufflesuite/truffle/pull/2350/checks?check_run_id=208414445#step:6:954

Reply:
Changed to the package name.